### PR TITLE
[INLONG-6996][Dashboard] Add JSON format for InlongStream

### LINKED ISSUE #6996: [Improve][Dashboard] Add JSON format for InlongStream
### Description

JSON has become popular as a data format for developers because of its human-readable text, which is lightweight, requires less coding, and processes faster.

image

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by featzhang**
### Prepare a Pull Request

- Fixes #6996 

### Motivation

Add JSON format for InlongStream.

### CODE REVIEW

**Comment by dockerzhang** [comment]
@ZuoFengZhang I think we have to make sure the Sort could know this data type.
Otherwise, do you miss the TubeMQ Operator?

**Comment by dockerzhang** [comment]
@featzhang please rebase from the master and fix the conflict.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java [hunk 1]
```diff
@@ -111,6 +111,8 @@ public Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
             kafkaSource.setSourceName(streamId);
             kafkaSource.setBootstrapServers(bootstrapServers);
             kafkaSource.setTopic(streamInfo.getMqResource());
+            kafkaSource.setSerializationType(streamInfo.getDataType());
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java:114]: Please rebase the master branch after the #7001 merging.

**Review by healchow — CHANGES_REQUESTED** [review]

**Review by leezng — APPROVED** [review]

**Comment by featzhang** [comment]
> @featzhang I think we have to make sure the Sort could know this data type. Otherwise, do you miss the TubeMQ Operator?

I have fixed it in another pr #6998

**Review by healchow — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**